README.md

```diff
@@ -60,7 +60,15 @@ We could retrieve this value from the repo settings through an API call but that
 
 Default: true
 
-The action will skip PRs that have failed checks.
+The action will skip PRs that have failed checks. Please note that if `allow_ongoing_checks` is not set to `true`, the action will skip PRs with ongoing checks. This may result in the failure to update PR branches when the action is triggered while checks for those PRs are still in progress.
```

Comment:
```suggestion
The action will skip updating PRs that have failed checks. Please note that if `allow_ongoing_checks` is set to `false`, the action will skip updating PRs with ongoing checks. This will result in the failure to update PR branches when the action is triggered while checks for those PRs are still in progress.
```

Reply:
thanks